tools/rosbag/src/recorder.cpp

```diff
@@ -129,6 +130,8 @@ Recorder::Recorder(RecorderOptions const& options) :
 }
 
 int Recorder::run() {
+    ROS_INFO("Running bag recorder");
```

Comment:
Should make a ticket to remove these that is blocked by the ticket to find the bagging crash issue so we don't forget to remove these.

Reply:
Will do.